examples/audit-report.html

```diff
@@ -0,0 +1,225 @@
+language: "en"
+pageclass: "oag-theme wb-prettify all-pre"
+layout: without-h1
+---
+
+<div class="container">
+	<div class="pb-4 pt-4 bg-center bg-cover" data-bgimg-srcset="https://wet-boew.github.io/vocab/wb/utilities#no-image 991w, ../assets/img/bear-background.png 992w">
+		<div class="row">
+			<div class="col-md-7">					
+					<h1 property="name" id="wb-cont">{% if page.language == "en" %}Public housing in Nunavut{% else %}Logements sociaux au Nunavut{% endif %}</h1>
+					<p>{% if page.language == "en" %}The introduction block pattern introduces the content of a landing page.{% else %}La configuration de conception du bloc d'introduction introduit le contenu d'une page de destination.{% endif %}</p>
+			</div>
+		</div>
+	</div>
+	<div class="row">
+		<div class="col-sm-4">
+			<section class="well">			
+				<h3 class="text-uppercase">Tabling date</h3>
+				<p class="small"><time datetime="2025-06-05"><span>May 6, 2025</span></time></p>
+				<h3 class="text-uppercase">Audited entities</h3>
+				<ul class="list-unstyled">
+					<li>Nunavut Housing Corporation</li>
+					<li>Government of Nunavut</li>
+				</ul>
+				<h3 class="text-uppercase">Topics</h3>
+				<ul class="list-unstyled">
+					<li>Air</li>
+					<li>Climate Change</li>
+					<li>Agriculture</li>
+					<li>Public Service</li>
+					<li>Taxation</li>
+				</ul>
+				<h3 class="text-uppercase">Related Audits</h3>
+				<ul class="list-unstyled">
+					<li>Reports to Northern Legislative</li>
+					<li>Assemblies</li>
+				</ul>
+
+			</section>
+			<a class="gc-dwnld-lnk" href="#">
+				<div class="well gc-dwnld">
+					<div class="row">
+						<div class="col-xs-12">
+							<p><img class="img-responsive thumbnail gc-dwnld-img" src="../assets/img/doc.png" alt="" /></p>
+						</div>
+						<div class="col-xs-12">
+							<p class="gc-dwnld-txt">
+								<span>Download Report</span>
+								<span class="gc-dwnld-info">(<abbr title="Portable Document Format">PDF</abbr>, 273 <abbr title="KiloByte">KB</abbr>)</span>
+							</p>
+						</div>
+					</div>
+				</div>
+			</a>
+			<div class="well">
+				<div class="row">					
+					<div class="col-xs-12">
+						<a href="#">Request another format </a>
+					</div>
+				</div>
+			</div>
+			<div class="well gc-dwnld">
+				<div class="row">					
+					<div class="col-xs-12">
+						<p class="gc-dwnld-txt">
+							<span class="gc-dwnld-info">Media Assets (273<abbr title="KiloByte">KB</abbr>)</span>
+						</p>
+					</div>
+				</div>
+			</div>
+			<div class="reports wb-eqht-grd hght-inhrt">
+				<section class="well oag-rds">
+					<hgroup class="mb-3">
+						<h3 class="label label-primary fnt-hdng">News Release</h3>
+						<p class="fw-bold small text-uppercase"><time datetime="2025-03-18"><span>April 18, 2025</span></time></p>
+					</hgroup>
+					<p class="mb-3">2025 Spring Reporst of the Auditor General of Canada and the Commisioner of the Environment and Sustainable Development - Opening Statement.</p>
+				</section>
```

Comment:
I feel like these should be `<article>` instead of `<section>` elements. They are literally articles.